src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -131,6 +139,30 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     group5: 4,
   };
 
+  const { enabled: adsEnabled } = useToggle('ads');
+  const { isAmp, showAdsBasedOnLocation } = useContext(RequestContext);
+
+  /**
+   * Should we display ads? We check:
+   * 1. The CPS `allowAdvertising` field value.
+   * 2. A value local to the STY page type.
+   * - iSite toggles are handled by the Ad container.
+   */
+  console.log(
+    '1 -->',
+    path(['metadata', 'options', 'allowAdvertising'], pageData),
+  );
+
+  console.log('2 -->', adsEnabled);
```

Comment:
Are these console.logs meant to be left in?

Reply:
@chris-hinds removed!